### DISCUSSION TITLE
Import via VHDX

### DISCUSSION THREAD

**Discussion by scottstephens** [nix-community/NixOS-WSL#193] [Ideas]
I noticed that (somewhat) recently the `wsl --import` command got a new `--vhd` argument, which imports a distribution from a vhdx file instead of a tarball.

Has anybody thought about whether this could eliminate the need for the installer workaround to the issues with hard links in tarballs?

**Comment by nzbr** [comment]
That might be possible, but I don't think it'd be compatible with the API calls used by the DistroLauncher (once that's merged), so it wouldn't eliminate the need for the installer completely. It might be an alternative for people who don't want to use the Launcher though
